e2e/resolve-with-paths/__tests__/resolveWithPaths.test.js

```diff
@@ -35,3 +35,12 @@ test('throws an error if the module cannot be found from given paths', () => {
     "Cannot resolve module './mod.js' from paths ['..'] from "
   );
 });
+
+test('throws module not found error if the module cannot be found from given paths', () => {
+  try {
+    require.resolve('./mod.js', {paths: ['..']});
+  } catch (err) {
+    expect(err.code).toBe('MODULE_NOT_FOUND');
+  }
+  expect.hasAssertions();
```

Comment:
At the start of the test, please

Might consider using `.rejects.toMatchObject({code: 'MODULE_NOT_FOUND'})` up to you :)